src/dtclient/kubernetes_settings_test.go

```diff
@@ -0,0 +1,389 @@
+package dtclient
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testUID      = "test-uid"
+	testName     = "test-name"
+	testObjectID = "test-objectid"
+	testScope    = "test-scope"
+)
+
+func createMonitoredEntities() []MonitoredEntity {
```

Comment:
This name could be confused with actual (not test) code, so naming like `createTestingMonitoredEntities ` or something similar